frontend/app/src/composables/balances/aggregated.ts

```diff
@@ -62,7 +62,7 @@ export const useAggregatedBalances = () => {
           return asset;
         }
       );
-      const assets = get(balances(hideIgnored)).map(({ asset }) => {
+      const assets = get(balances(hideIgnored, false)).map(({ asset }) => {
```

Comment:
fix issue where the prices for asset that under GROUPED_MULTI_CHAIN doesn't get fetch